multisig/proposals/arbitrumsepolia/vip-003/index.ts

```diff
@@ -0,0 +1,203 @@
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS_BRIDGE_ADMIN_PROXY, "updateSendAndCallEnabled(bool)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "mint(address,uint256)", XVS_BRIDGE_DEST],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "burn(address,uint256)", XVS_BRIDGE_DEST],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS_BRIDGE_ADMIN_PROXY, "setTrustedRemoteAddress(uint16,bytes)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS_BRIDGE_ADMIN_PROXY, "transferBridgeOwnership(address)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "migrateMinterTokens(address,address)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "setMintCap(address,uint256)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "updateBlacklist(address,bool)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "pause()", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "unpause()", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    { target: XVS_BRIDGE_ADMIN_PROXY, signature: "acceptOwnership()", params: [] },
+    {
+      target: XVS_BRIDGE_ADMIN_PROXY,
+      signature: "setTrustedRemoteAddress(uint16,bytes)",
+      params: [BNB_TESTNET_ENDPOINT_ID, BNB_TESTNET_TRUSTED_REMOTE],
+    },
+    {
+      target: XVS_BRIDGE_ADMIN_PROXY,
+      signature: "setTrustedRemoteAddress(uint16,bytes)",
+      params: [OPBNB_TESTNET_ENDPOINT_ID, OPBNB_TESTNET_TRUSTED_REMOTE],
+    },
+    {
+      target: XVS_BRIDGE_ADMIN_PROXY,
+      signature: "setTrustedRemoteAddress(uint16,bytes)",
+      params: [SEPOLIA_ENDPOINT_ID, SEPOLIA_TRUSTED_REMOTE],
+    },
+    {
+      target: XVS_BRIDGE_ADMIN_PROXY,
+      signature: "setWhitelist(address,bool)",
+      params: [arbitrumsepolia.VTREASURY, true],
+    },
+    {
+      target: XVS_BRIDGE_ADMIN_PROXY,
+      signature: "setWhitelist(address,bool)",
+      params: [arbitrumsepolia.NORMAL_TIMELOCK, true],
+    },
+    {
+      target: XVS,
+      signature: "setMintCap(address,uint256)",
+      params: [XVS_BRIDGE_DEST, XVS_MINT_LIMIT],
+    },
```

Comment:
Don't we need the commands to configure the bridge limits? as we did here for Ethereum, for example: https://github.com/VenusProtocol/vips/blob/develop/multisig/proposals/ethereum/vip-014/index.ts

We can start with these limits:

Limit | Amount (USD)
-- | --
MaxSingleTransactionLimit | 10,000
MaxDailyLimit | 50,000
MaxSingleReceiveTransactionLimit | 10,200
MaxDailyReceiveLimit | 51,000

We would need to set it for each peer network: bsctestnet, sepolia and opbnb testnet

Reply:
These commands were executed during the deployment of the bridge as we inlcude these commands [here](https://github.com/VenusProtocol/token-bridge/pull/25/files#diff-18185c86bd8baed36466d7a325efd11e7729ffdd43200a4808db76a6340185ffR152).
You can see the txs [here](https://sepolia.arbiscan.io/address/0xFdC5cEC63FD167DA46cF006585b30D03B104eFD4).

If we have changed the values of `MaxSingleReceiveTransactionLimit` and `MaxDailyReceiveLimit` now, I will include these commands to set the new values.

Other than that I think we should also update the values of `MaxSingleReceiveTransactionLimit` and `MaxDailyReceiveLimit` in the bridge PR too to keep the updated data and perform same txs for **Arbitrum One** with correct values.

Reply:
Done https://github.com/VenusProtocol/vips/pull/249/commits/e88d273c9f35b498b210078d704a5be113fb0daa

Reply:
https://github.com/VenusProtocol/token-bridge/pull/25/files#r1577659293